app/models/email_subscription.rb

```diff
@@ -4,11 +4,42 @@ class EmailSubscription < ApplicationRecord
   validates :name, presence: true
   validates :topic_slug, presence: true
 
+  before_destroy :deactivate!
+
   def to_hash
     {
       "name" => name,
       "topic_slug" => topic_slug,
       "email_alert_api_subscription_id" => email_alert_api_subscription_id,
     }.compact
   end
+
+  def reactivate_if_confirmed!(email, email_verified)
+    deactivate!
+
+    return unless email_verified
+
+    subscriber_list = GdsApi.email_alert_api.get_subscriber_list(
+      slug: topic_slug,
+    )
+
+    subscription = GdsApi.email_alert_api.subscribe(
+      subscriber_list_id: subscriber_list.to_hash.dig("subscriber_list", "id"),
+      address: email,
+      frequency: "daily",
```

Comment:
Are we happy with "daily" being the default here?
Gavin & co might be worth checking in with, I think there was some work on notifications previously to make a weekly digest the default as there were complaints from users about getting too many emails from default settings?

Not sure if that's still a valid concern here?

Reply:
Daily is what we currently use for the transition checker subscriptions, so I just copied that over